app.R

```diff
@@ -1,6 +1,10 @@
-app_version_choices <- jsonlite::fromJSON(Sys.getenv("APP_VERSION_CHOICES", "[\"dev\"]"))
+app_version_choices <- jsonlite::fromJSON(Sys.getenv(
+  "APP_VERSION_CHOICES",
+  "[\"dev\"]"
+))
 
-"%||%" <- function(x, y) { # nolint
+"%||%" <- function(x, y) {
+  # nolint
```

Comment:
Something to be cognizant of when we autoformat!

```suggestion
"%||%" <- function(x, y) { # nolint
```

Reply:
oh, damn. yes!

Reply:
there is an issue air (256) about this. I'm going to switch to nolint start/end